### PR TITLE
Multisuite support in adjudicator

### PR DESCRIPTION
**Pull request by agoddard**
teapot and serverspec verdict & metadata are now aware of instances
testing is now run against both failed payloads and successful payloads

### CODE REVIEW

**Comment by agoddard** [comment]
:key:

File: lib/jackal-kitchen/adjudicate.rb [hunk 1]
```diff
@@ -123,9 +127,19 @@ def populate_reasons_for_failure(payload)
       #
       # @param payload [Smash] payload data with test example info
       # @return [Smash] metadata associated with test type
-      def metadata(payload, type)
+      def metadata(data, type)
         meth = (type == :teapot) ? :teapot_metadata : :spec_metadata
-        send(meth, payload.get(:data, :kitchen, :test_output, type))
+        send(meth, data)
+      end
+
```
Comment (chrisroberts) [inline lib/jackal-kitchen/adjudicate.rb:134]: :scroll: (yardoc)

File: lib/jackal-kitchen/adjudicate.rb [hunk 2]
```diff
@@ -123,9 +127,19 @@ def populate_reasons_for_failure(payload)
...
+      def test_output(payload, *args)
+        payload.get(:data, :kitchen, :test_output, *args)
+      end
+
```
Comment (chrisroberts) [inline lib/jackal-kitchen/adjudicate.rb:142]: :scroll:

**Comment by agoddard** [comment]
:scroll: 'd

**Pull request merged**